tests/constants.py

```diff
@@ -95,7 +127,7 @@ def test_py3kwarning_of_sys(self):
     self.assertOnlyIn((2, 6), detect("from sys import py3kwarning"))
 
   def test_subversion_of_sys(self):
-    self.assertOnlyIn((2, 5), detect("from sys import subversion"))
+    self.assertOnlyIn(((2, 5), (3, 0)), detect("from sys import subversion"))
```

Comment:
Yes, because vermin is not considering about feature **deprecation** (expect Python 2 vs 3 compatibility), only checking the version that new features are introduced. I guess the intended users of vermin is those who use a new version of Python for development but expect their code to run on older versions of Python, so they can notice deprecation warnings and even feature removals by themselves during development.

Reply:
That is a feature I have on my long list of improvements, though. Yes, this is the intended audience - more or less.